### PR TITLE
Fix add / remove buttons once displayed the menu.

### PR DESCRIPTION
**Pull request by raidercillo**

### CODE REVIEW

File: sample/res/values/strings.xml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
 	<string name="app_name">Button Menu</string>
+    <string name="button_text_remove_add_button">Remove/Add moment ButtonVM</string>
```
Comment (pedrovgs) [inline sample/res/values/strings.xml]: Wrong indentation, can you fix this?

File: sample/res/layout/activity_second.xml [hunk 2]
```diff
@@ -44,6 +50,7 @@
 
 	<com.tuenti.buttonmenu.ButtonMenu
 			android:id="@+id/button_menu"
+            android:weightSum="555"
```
Comment (pedrovgs) [inline sample/res/layout/activity_second.xml]: Wrong indentation, can you fix this?

File: sample/res/layout/activity_second.xml [hunk 3]
```diff
@@ -8,6 +8,12 @@
...
 
+        <Button
+            android:id="@+id/bt_remove_add_button"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:text="@string/button_text_remove_add_button"/>
+
 		<Button
 				android:id="@+id/bt_enable_disable_all"
 				android:layout_width="match_parent"
```
Comment (pedrovgs) [inline sample/res/layout/activity_second.xml]: Wrong indentation, can you fix this?

**Comment by pedrovgs** [comment]
Awesome pull request, can you update the gif used in the README.md to show this new functionality? Another question, do you need a public release with this feature?

**Comment by pedrovgs** [comment]
Be careful with the gif size! Review the current one before to record a new one, please :)

**Comment by pedrovgs** [comment]
@oscarsj we are going to need a public release of this library to Maven Central once thsi PR be merged. FYI

**Comment by raidercillo** [comment]
Thanks! I'll correct the pending issues and provide a new screenshot for the new UI of the sample app!

**Comment by pedrovgs** [comment]
Thanks for your PR @raidercillo. We are going to merge this pull request and publish a new version in Maven Central ;)

**Comment by pedrovgs** [comment]
@oscarsj or @jsoriano can you merge this PR?

**Pull request merged**